lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -175,6 +175,7 @@
     - '     - C(eagerzeroedthick) eagerzeroedthick disk, added in version 2.5'
     - '     Default: C(None) thick disk, no eagerzero.'
     - ' - C(datastore) (string): Datastore to use for the disk. If C(autoselect_datastore) is enabled, filter datastore selection.'
+    - ' - C(filename) (string): Existing disk image to use. Must already exist on the datastore. String is in C([datastore_name] path/to/file.vmdk) format'
```

Comment:
Specifying the datastore in the string is redundant to the `C(datastore)` field.  Perhaps the filename should just be the path to the file without the `[ datastore_name ]` part?

Reply:
Alternatively, since vmdk_disk_path_split separates out the datastore by name, one should be able to specify the filename by itself.  I'd be okay with that too.